diff --git a/config/m4/java.m4 b/config/m4/java.m4
index e624519f4..42302ee55 100644
--- a/config/m4/java.m4
+++ b/config/m4/java.m4
@@ -14,6 +14,10 @@ AC_ARG_WITH([java],
                             [Compile Java UCX (default is guess).])
             ], [], [with_java=guess])
 
+echo "*********************************************"
+java -version
+echo "with_java: $with_java"
+echo "*********************************************"
 AS_IF([test "x$with_java" != xno],
       [
        AC_CHECK_PROG(MVNBIN,  mvn,  yes)
